lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -123,6 +124,18 @@ class Text3D(mtext.Text):
 
     def __init__(self, x=0, y=0, z=0, text='', zdir='z', axlim_clip=False,
                  **kwargs):
+        if kwargs.get('rotation', None) is not None:
+            warnings.showwarning(
```

Comment:
you may want to use
```suggestion
            _api.warn_external(
```

https://matplotlib.org/stable/api/_api_api.html#matplotlib._api.warn_external

Reply:
Done